match/lib/match/git_helper.rb

```diff
@@ -48,25 +46,19 @@ def self.match_version(workspace)
     end
 
     def self.commit_changes(path, message, git_url)
-      Dir.chdir(path) do
-        return if `git status`.include?("nothing to commit")
```

Comment:
Any reason why we got rid of this line? Will it behave fine if there were no changes?